moveit_core/trajectory_processing/test/test_time_optimal_trajectory_generation.cpp

```diff
@@ -150,6 +150,247 @@ TEST(time_optimal_trajectory_generation, test3)
   EXPECT_DOUBLE_EQ(90.0, trajectory.getPosition(trajectory.getDuration())[3]);
 }
 
+TEST(time_optimal_trajectory_generation, testLargeAccel1)
+{
+  double path_tolerance = 0.1;
+  double resample_dt = 0.1;
+  Eigen::VectorXd waypoint(6);
+  std::list<Eigen::VectorXd> waypoints;
+  Eigen::VectorXd max_velocities(6);
+  Eigen::VectorXd max_accelerations(6);
+  waypoint << 1.6113056281076339, -0.21400163389235427, -1.974502599739185, 9.9653618690354051e-12, -1.3810916877429624,
```

Comment:
use `// clang-format off` and `// clang-format on` when clang format makes things less readable such as for these. 
Also, do you need this input trajectory to be so long? It is hard to tell where in the input is causing the bug..

Reply:
Both test cases trigger the same bug, should we pick just the short one .?

Reply:
I forget to mention that the `waypoints` of the short test is a subset of the `waypoints` of the long test

Reply:
Yes, let's just keep the shorter one. No need for the extra complexity unless it adds additional coverage.